sgt-back-end/server.ts

```diff
@@ -0,0 +1,130 @@
+import pg from 'pg';
+import express from 'express';
+import { ClientError } from './client-error.js';
+import { errorMiddleware } from './error-middleware.js';
+
+// only create ONE pool for your whole server
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+app.use(express.json());
+app.listen(8080, () => console.log('app is listenting on port 8080'));
+
+app.get('/api/grades', async (req, res, next) => {
+  try {
+    const sql = `
+      select *
+        from "grades"
+    `;
+    const result = await db.query(sql);
+    const grade = result.rows;
```

Comment:
There are multiple grades, so the name of this variable should probably be plural.